verification.py

```diff
@@ -1269,30 +1263,40 @@ def _msms_params_to_run_msp(self, params):
                     position=params["sel_pos"],
                     end_frequency=params["end_frequency_lst"][i],
                     start_frequency=0.5 / params["refsize"],
-                    alpha=params["alpha"],
+                    s=params["alpha"] / params["refsize"],  # alpha=saA, s=sAA/(2N)
                     dt=1.0 / (40 * params["refsize"]),
                 )
                 t_start = params["end_time_lst"][i]
                 model.append((t_start, temp_model))
-            model.append((None, None))
+            model.append((None, "hudson"))
 
         else:
             model = "hudson"
 
-        scale_factor = params["num_sites"]
-        recombination_rate = params["rho"] / (scale_factor - 1)
+        scale_factor = 4.0 * params["refsize"] * (params["num_sites"] - 1)
+        recombination_rate = params["rho"] / scale_factor
+        scale_factor = 4.0 * params["refsize"] * params["num_sites"]
         mutation_rate = params["theta"] / scale_factor
 
-        ts = msprime.simulate(
-            sample_size=params["nsam"],
-            Ne=0.25,
-            length=params["num_sites"],
-            mutation_rate=mutation_rate,
-            recombination_rate=recombination_rate,
+        repeats = msprime.sim_ancestry(
+            samples=params["nsam"],
+            population_size=params["refsize"],
             model=model,
+            recombination_rate=recombination_rate,
+            discrete_genome=False,
+            sequence_length=params["num_sites"],
+            num_replicates=params["nrep"],
         )
 
-        return ts
+        # Critical to use BinaryMutationModel and get ancestral and derived alleles
+        mutated_repeats = [
+            msprime.sim_mutations(
+                ts, rate=mutation_rate, model=msprime.BinaryMutationModel()
```

Comment:
Need discrete_genome=False here too @gbinux

Reply:
Oops, missed the one in sim_mutations. Just added it as well. @jeromekelleher 

[MsmsSweeps_2.zip](https://github.com/tskit-dev/msprime/files/6175848/MsmsSweeps_2.zip)